src/notation/view/widgets/selectnotedialog.cpp

```diff
@@ -234,39 +234,38 @@ void SelectNoteDialog::apply() const
         return;
     }
 
-    EngravingItem* selectedElement = interaction->selection()->element();
+    EngravingItem* selectedElement = contextItem(interaction);
     if (!selectedElement) {
         return;
     }
 
-    FilterElementsOptions options = noteOptions();
```

Comment:
in reference-based languages (C#/Java etc.) this would have worked fine, but in C++ it's a slice!